config/settings/base.py

```diff
@@ -43,6 +43,13 @@
 DATABASES = {"default": env.db("DATABASE_URL")}
 DATABASES["default"]["ATOMIC_REQUESTS"] = True
 
+
+# ELASTICSEARCH
+# ------------------------------------------------------------------------------
+ELASTICSEARCH_DSL = {
+    "default": {"hosts": env("ELASTICSEARCH_URL")},
```

Comment:
why dict with default?

Reply:
According to docs [Here](https://django-elasticsearch-dsl.readthedocs.io/en/latest/quickstart.html)
I found that it must be defined